### ISSUE TITLE
Meaning of "adam1e-4-tunelr"

### ISSUE THREAD

**Issue by ayst123** [meteorshowers/RCF-pytorch#4]
Thanks for this repo.  I am not clear about one thing in result part.  What is meaning of "adam1e-4-turnlr" ?  If we use adam as the optimizer, do we need to turn learning rate during traning ?  

Thanks.

**Comment by shiyanrubing** [comment]
In my view, 1e-4 should be initial learning rate.

**Comment by meteorshowers** [comment]
sorry for the late response~
yes, this is the learning rate
